app/policies/claim_policy.rb

```diff
@@ -11,6 +11,14 @@ def submit?
     !user.support_user? && !record.submitted?
   end
 
+  def destroy?
+    user.support_user?
```

Comment:
You could additionally ensure that the claim is in one of the states which will allow you do perform this action.
```suggestion
    user.support_user? && (record.submitted? || record.draft?)
```